### DISCUSSION TITLE
Pricing

### DISCUSSION THREAD

**Discussion by fungamertyyy111** [Akascape/CTkDesigner-Support#30] [Q&A]
What is this good tool not free, I am a broke 14y/o boy trying to make a pretty complex app.

**Comment by Akascape** [comment]
@fungamertyyy111 This tool is actually for those users who want to save time making UI and add features in one click.

I recommend you learn how to code the UI with customtkinter instead, learn tkinter and customtkinter, go through my posts and free material which you can find on my github and website, then start making your app. 
CTkDesigner took a lot of manual effort and time. It is targeted for commercial purpose only.
I hope you understand :)
